database/sql/src/main/scala/cromwell/database/slick/MetadataSlickDatabase.scala

```diff
@@ -406,4 +409,8 @@ class MetadataSlickDatabase(originalDatabaseConfig: Config)
     )
   }
 
+  override def queryRootWorkflowSummaryEntriesByArchiveStatusAndOlderThanTimestamp(archiveStatus: Option[String], thresholdTimestamp: Timestamp)(implicit ec: ExecutionContext): Future[Seq[WorkflowMetadataSummaryEntry]] = {
```

Comment:
Against which database would it make sense to run this test?

Reply:
I think @aednichols has a (relatively recent?) clone of the production database. If you can work out what the SQL query would look like from this and then try running it against that (probably manually), I think that would be a good sanity check that the query will be performant.

Reply:
Result from prod: `select * from WORKFLOW_METADATA_SUMMARY_ENTRY where metadata_archive_status = 'Archived' and root_workflow_execution_uuid is null and parent_workflow_execution_uuid is null and end_timestamp < date_sub(now(), interval 24 hour);
Empty set (0.09 sec)`

Also looking at the query plan, it shows that query with `metadata_archive_status` condition will use index IX_WORKFLOW_METADATA_SUMMARY_ENTRY_MAS, and otherwise it's a full scan.